pkgs/applications/graphics/ktikz/default.nix

```diff
@@ -30,10 +29,12 @@ stdenv.mkDerivation rec {
     sha256 = "1s83x8r2yi64wc6ah2iz09dj3qahy0fkxx6cfgpkavjw9x0j0582";
   };
 
-  patches = [ (fetchurl {
-    url = "https://github.com/fhackenberger/ktikz/commit/972685a406517bb85eb561f2c8e26f029eacd7db.patch";
-    sha256 = "16jwsl18marfw5m888vwxdd1h7cqa37rkfqgirzdliacb1cr4f58";
-  })];
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/fhackenberger/ktikz/commit/972685a406517bb85eb561f2c8e26f029eacd7db.patch";
+      sha256 = "16jwsl18marfw5m888vwxdd1h7cqa37rkfqgirzdliacb1cr4f58";
```

Comment:
You will also need to update the hash, since the file is a different fixed output derivation now.

Reply:
Beaten, once again, by nix accepting existing derivations with existing hash without checking.